### DISCUSSION TITLE
Drag and Drop not working, v 4.22

### DISCUSSION THREAD

**Discussion by whwkong** [i3/i3#5439] [Q&A] [answered]
For the blissful months since the i3 4.21 release, drag and drop for tiling windows worked fine.

Recently I updated to 4.22, and suspiciously around that time, my drag and drop stopped working.

AFAIK, I have not modified my i3 config substantially since ... 

I'm not sure what to do to get my drag and drop working again.

**Comment by orestisfl** [comment, marked as answer]
This discussion is not really actionable as it is, please open a proper bug request in a new issue including logs of a new i3 session where you just login, open a couple of windows and drag one of them.

**Comment by whwkong** [comment]
My bad .. it must be something in my config file --- the default config works.
